<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="line_nama">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_alamat">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_email">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nama</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Alamat</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Pendidikan</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_pendidikan">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="save_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_prestasi1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_prestasi2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Prestasi 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_prestasi3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Perstasi 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Prestasi 1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_ipk">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>GPA / IPK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="foto_btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ambil foto profil</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
